datacontract/templates/partials/model_field.html

```diff
@@ -111,4 +111,12 @@
 
 {% if field.items %}
 {{ render_nested_partial("item", field.items, level) }}
-{% endif %}
\ No newline at end of file
+{% endif %}
+
+{% if field.keys %}
+{{ render_nested_partial("key", field.keys, level) }}
```

Comment:
I'd rather use "keys" and "values"

Reply:
Also align arrays?

<img width="570" alt="Bildschirmfoto 2024-07-22 um 23 25 19" src="https://github.com/user-attachments/assets/405f43d8-0781-4f8f-9aab-5cae9effb988">